crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1196,8 +1196,26 @@ impl World {
     pub fn is_resource_added<R: Resource>(&self) -> bool {
         self.components
             .get_resource_id(TypeId::of::<R>())
-            .and_then(|component_id| self.storages.resources.get(component_id)?.get_ticks())
-            .map(|ticks| ticks.is_added(self.last_change_tick(), self.read_change_tick()))
+            .map(|component_id| self.is_resource_added_by_id(component_id))
+            .unwrap_or(false)
+    }
+
+    /// Return's `true` if a resource with id `component_id` exists and was added since the world's
+    /// [`last_change_tick`](World::last_change_tick()). Otherwise, this return's `false`.
+    ///
+    /// This means that:
+    /// - When called from an exclusive system, this will check for additions since the system last ran.
+    /// - When called elsewhere, this will check for additions since the last time that [`World::clear_trackers`]
```

Comment:
I would change "this" to "it", feels easier to read.

Reply:
Not sure how I feel about that, what do other people think?

Reply:
This really is a nitpick and nothing relevant.

Reply:
Oh, totally just a nitpick! I just felt like it would be easier to read, but it might just be me.

Reply:
I prefer this :p